tests/smoketest/smoketest_collectd_entrypoint.sh

```diff
@@ -65,6 +65,9 @@ if [ "$DOCUMENT_HITS" -gt "0" ]; then
     events_result=0
 fi
 
+echo "[INFO] Verification results: events=${events_result} metrics=${metrics_result}"
```

Comment:
Same as above. I know the variables are called `_result`, which is fine in the context of a shell script, but for output purposes it would be nice to clearly call these out as exit codes.